#For NTU VIRAL dataset, using sensors: 

ilive_common:
   if_dump_log: 0                   # If recording ESIKF update log. [default = 0]
   record_offline_map: 1            # If recording offline map. [default = 1]
   pub_pt_minimum_views: 3          # Publish points which have been render up to "pub_pt_minimum_views" time. [default = 3]
   minimum_pts_size: 0.01           # The minimum distance for every two points in Global map (unit in meter). [default = 0.01] 
   image_downsample_ratio: 1        # The downsample ratio of the input image. [default = 1]
   estimate_i2c_extrinsic: 0        # If enable estimate the extrinsic between camera and IMU. [default = 1]
   estimate_intrinsic: 0            # If enable estimate the online intrinsic calibration of the camera lens. [default = 1] 
   maximum_vio_tracked_pts: 600     # The maximum points for tracking. [default = 600]
   append_global_map_point_step: 4  # The point step of append point to global map. [default = 4]
   gravity: [ 0 ,0, 9.805]       # Gravity
   esikf_iter_f2f_times: 1
   esikf_iter_f2map_times: 1
   use_lio: 1
   use_vio_f2f: 0
   use_vio_f2map: 0
   use_consistent_solution: true
   if_save_mapping: false
   method: "EKF"                         #EKF,InEKF,EIKF


#here we use cam0

ilive_vio:
   image_width: 752
   image_height: 480
   camera_intrinsic:
      [431.3873911369959,0, 427.4407802012019,
      0.0,  430.2496176152663, 238.52694867508183,
      0.0, 0.0, 1.0 ] 
   camera_dist_coeffs: [0.007017508922248575, -0.002598448110748391, -0.0014755213248257705, 0.0006640775125219394, 0]  #k1, k2, p1, p2, k3
   
   # <imu to cam0>
   camera_ext_R:
         [0.70992163, -0.70414167,  0.01399269,
          0.02460003,  0.00493623 ,-0.99968519 ,
         0.70385092 , 0.71004236 , 0.02082624]
   camera_ext_t: [ 0.01562753, -0.01981648, -0.07544143]




Lidar_front_end:
   lidar_type: 3   # 0 for mid, 1- Livox-horizon, 2 for velodyne 16, ï¼Œ3 for Ouster
   N_SCANS: 64
   using_raw_point: 1
   point_step: 1


ilive_lio:        
   lio_update_point_step: 4   # Point step used for LIO update.
   max_iteration: 2         # Maximum times of LIO esikf. The number of the iterative steps.//1.2.3
   lidar_time_delay: 0.015        # The time-offset between LiDAR and IMU, provided by user. 
   filter_size_corner: 0.30   
   filter_size_surf: 0.30
   filter_size_surf_z: 0.30
   filter_size_map: 0.30

   ext_t_lid_in_imu: [-0.2240, -0.1307, 0.1369] 
   ext_R_lid_in_imu:
              [   -0.9998  ,  0.0023,   -0.0190,
   -0.0026,   -0.9999  ,  0.0131,
   -0.0190  ,  0.0132   , 0.9997]
   publish_feature_map: true  
   long_rang_pt_dis: 1000              
   planar_check_dis: 0.2                                                                           



#use IMU in Ouster64
ilive_IMU:
   max_IMU_calibration_num: 100
   COV_ACC_NOISE_DIAG:  1.560001e-4  #discerete
   COV_OMEGA_NOISE_DIAG: 4.3264e-6  #discrete
   COV_BIAS_ACC_NOISE_DIAG: 1.1236e-11
   COV_BIAS_GYRO_NOISE_DIAG: 1.6e-13